services/src/test/java/org/keycloak/protocol/oid4vc/model/VerifiableCredentialTest.java

```diff
@@ -0,0 +1,75 @@
+package org.keycloak.protocol.oid4vc.model;
```

Comment:
When creating a new class file, could you add the following header comments on top of the file?

```
/*
 * Copyright 2024 Red Hat, Inc. and/or its affiliates
 * and other contributors as indicated by the @author tags.
 *
 * Licensed under the Apache License, Version 2.0 (the "License");
 * you may not use this file except in compliance with the License.
 * You may obtain a copy of the License at
 *
 * http://www.apache.org/licenses/LICENSE-2.0
 *
 * Unless required by applicable law or agreed to in writing, software
 * distributed under the License is distributed on an "AS IS" BASIS,
 * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 * See the License for the specific language governing permissions and
 * limitations under the License.
 */
```

Reply:
done